src/app/(auth)/signup/page.tsx

```diff
@@ -38,10 +42,17 @@ const Signup = () => {
     },
   });
   const { classes } = useStyles();
-  const { mutate } = trpc.auth.signup.useMutation();
+  const { mutate, error } = trpc.auth.signup.useMutation({
+    onSuccess: (data) => {
+      setUserData(data);
+    },
+  });
+
+  // To check if there is no input error
+  const isDisabled = Object.keys(form.errors).length !== 0 ? true : false;
 
-  if (session) {
-    redirect("/");
+  if (userData) {
+    redirect("/login");
   }
```

Comment:
@Lada496 The reason for redirecting `/login` here is to navigate to the main page (`/`) through login page? 👀

Reply:
yeah, I thought it was better to navigate users to `/login` instead of `/` because this signup operation does not allow users to log in.

Reply:
@Lada496 I see, that makes sense!